env/io_posix.cc

```diff
@@ -724,6 +727,22 @@ IOStatus PosixRandomAccessFile::MultiRead(FSReadRequest* reqs,
       }
 
       req_wrap = static_cast<WrappedReadRequest*>(io_uring_cqe_get_data(cqe));
+      // Reset cqe data to catch any stray reuse of it
+      ((struct io_uring_cqe*)cqe)->user_data = 0xd5d5d5d5d5d5d5d5;
+      // Check that we got a valid unique cqe data
+      auto wrap_check = wrap_cache.find(req_wrap);
```

Comment:
Note: Any efficiency we were saving by using autovector is more than lost by using an unordered set to flag items. But probably fine.

Reply:
Yeah. I'm hoping this will be temporary and can be removed once the problem is root caused.

Reply:
We could use something like a bitmap is the performance is a concern. I guess we are OK about it for now.